docs/load_tests/quick_tests/2022-03-22/Load-test-2022-03-22.md

```diff
@@ -0,0 +1,113 @@
+
+### JVB metrics
+
+| METRIC                | FIRST JVB                                                   | SECOND JVB
+| --------------------- | ----------------------------------------------------------- | --------------------------
+| Received bandwidth    | ![recv bandwidth jvb 1](resources/bandwidth_rcv_jvb_1.png)  | ![recv bandwidth jvb 2](resources/bandwidth_rcv_jvb_2.png)
+| Transmitted bandwidth | ![tx bandwidth jvb 1](resources/bandwidth_tx_jvb_1.png)     | ![tx bandwidth jvb 2](resources/bandwidth_tx_jvb_2.png)
+| Received packets      | ![recv packets jvb 1](resources/packets_rcv_jvb_1.png)      | ![recv packets jvb 2](resources/packets_rcv_jvb_2.png)
+| Transmitted packets   | ![tx packets jvb 1](resources/packets_tx_jvb_1.png)         | ![tx packets jvb 2](resources/packets_tx_jvb_2.png)
+| CPU usage             | ![cpu usage jvb 1](resources/cpu_usage_jvb_1.png)           | ![cpu usage jvb 2](resources/cpu_usage_jvb_2.png)
+| CPU throttling        | ![cpu throttling jvb 1](resources/cpu_throttling_jvb_1.png) | ![cpu throttling jvb 2](resources/cpu_throttling_jvb_2.png)
+| Memory usage          | ![memory usage jvb 1](resources/memory_usage_jvb_1.png)     | ![memory usage jvb 2](resources/memory_usage_jvb_2.png)
+
+### Metrics of other components
+
+| METRIC                   | CHART
+| ------------------------ | -------------------------------------------------------------------
+| Jicofo CPU usage         | ![jicofo cpu usage](resources/jicofo_cpu_usage.png)
+| Prosody CPU usage        | ![prosody cpu usage](resources/prosody_cpu_usage.png)
+| Prosody memory usage     | ![prosody memory usage](resources/prosody_memory_usage.png)
+| Jitsi-front CPU usage    | ![jitsi front cpu usage](resources/jitsi_front_cpu_usage.png)
+| Jitsi-front memory usage | ![jitsi front memory usage](resources/jitsi_front_memory_usage.png)
+
+---
+
+## Interpretation of results
+
+At 15:35:30, the second JVB pod is ready, but there is no load balancing when the JVB is ready.
+This is explained by the fact that participants have already joined their conferences, and were assigned to the first JVB.
+The human participant was assigned to the second JVB because he joined his meeting when the second JVB was ready.
+
+---
+
+## Conclusion
+
+The HPA is working and we need approximatively 5 minutes to have a second JVB, further investigations will be needed to optimize the scaling of JVB pods.
+
+Once a participant is connected to a conference and assigned to a JVB, he will not be moved to another one unless he disconnects and connects again.
+Having this behaviour, if a JVB is overloaded, creating other JVB will not balance the load on all instances.
+Newly created JVB will only prevent new participant to be assigned to an overloaded JVB.
+Thus we can not reduce the load on a JVB unless some participants who are assigned to it leave their meetings.
+This is why it is very important to keep a low load on each JVB pod.
```

Comment:
Always have some pods (and therefore nodes)  in advance?

Reply:
If we want to connect 2,000 participants in 2mins when there is only one JVB, the HPA will schedule some pods, but they will not be running in time as it takes more than 2mins to schedule a node and launch a JVB pod on it. All participants will be assigned to the first JVB and the quality of the meeting will drastically decrease.

Have some JVB in reserve is a solution, but with octo activated, we do not really know what the load distribution will be on all JVB.